### ISSUE TITLE
Update Fee GL Codes for 3 and Remove Inactive Fee code GL numbers for inactive fees (11/29 Sprint) (LP) (Ready for UAT on PROD)

### ISSUE THREAD

**Issue by Nadin-Nader** [cityofaustin/atd-data-tech#14697]
##  Update the GL codes for the 3 fees listed below and update the Active Flag of inactive fees

**UPDATE GL ON THESE FEES (with all the 0's):**

FEECODE | FEEDESC | New GL
-- | -- | --
50311 | TURP - Usage Fee - Sidewalk/Behind Curb | 5120-6200-9400-4062
50174 | ROW Technology Surcharge | 5120-6200-9301-4366
83442 | SCP Node Support Pole Application Fee | 5120-6200-9400-4176

**Update the Active Flag:**

FEECODE | FEEDESC
-- | --
50330 | Investigation Fee (No Permit)
50331 | ROW Investigation Fee (Violation of permit conditions)
50332 | ROW Investigation Fee (Improper Advance Warning Sign)
50333 | ROW Investigation Fee (Improper Use of Device)
50334 | ROW Investigation Fee (Failure to correct Deficiency)
50335 | ROW Investigation Fee (Restricting Traffic during Peak Hours)
50340 | Application Fee
50343 | Permit Fee
50349 | Driveway Inspection Fee  - Transportation
50350 | Sidewalk Inspection Fee  - Transportation
50351 | Curb/Gutter Inspection Fee - Transportation
50353 | Concrete Re-inspection fee - Transportation
50095 | ATD Service Permit
50125 | Sound Permit Fee
50140 | Gate Receipts Fee
50149 | Excavation Inspection Fee - Transportation
50171 | Annual Fee
50186 | Residential Parking Decal
50187 | Residential Parking  Hang Tag
50218 | Barricading Fee
51284 | Bike Map Fee
51285 | Traffic Impact Analysis
51291 | Street Banners Fee
51293 | Open Records Requests Fee
50359 | Barricade Inspection Fee
50360 | ATD - After Hours Inspection and Review Fee
50361 | Barricade Impoundment Fee (Labor, includes vehicle and fuel)
50362 | Barricade Impoundment Fee (Storage)
50363 | Barricade Impoundment Fee (Posting of Legal Notices)
50364 | Lamppost Banner Fee
50365 | ATD Sign Review Fee
50366 | ATD Sign Fee
50369 | Addnt'l driveway Inspection fee - Transportation
83401 | AE - SCP Network Node Additional Application Fee
83402 | SCP Network Node Application Fee - ATD
83403 | ATD - SCP Network Node Additional Application Fee
83406 | ATD - SCP Annual ROW Rental Fee
83407 | ATD - SCP Node Support Pole Application Fee
83408 | AE - SCP Node Support Pole Application Fee
83410 | AE - SCP Transport Facility Application Fee
83439 | SCP Network Node Application Fee - Other
83450 | ATD - SCP Network Node Application Fee - ATD
83454 | ATD - SCP Network Node Application Fee - AE
83458 | ATD - SCP Network Node Application Fee - Other
83462 | ATD - SCP Transport Facility Application Fee
83469 | ATD - SCP Transport Facility ROW Usage Fee

</body></html></div>

**Comment by Nadin-Nader** [comment]
FEECODES:

50311	TURP - Usage Fee - Sidewalk/Behind Curb	5120-6200-9400-4062-0000-0000-0000-0000-0000
50174	ROW Technology Surcharge	                        5120-6200-9301-4366-0000-0000-0000-0000-0000
83442	SCP Node Support Pole Application Fee	5120-6200-9400-4176-0000-0000-0000-0000-0000

**Comment by TracyLinder** [comment]
@Nadin-Nader i updated the AC with all the fee info. This is ready to go.

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Committed changes , Ready for Dev deployment

**Comment by Nadin-Nader** [comment]
Waiting on #14112 to go first

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been sent to the DevOps Team to be deployed to DEV under **SCTASK0373960**

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT This has been deployed to DEV please UAT

**Comment by TracyLinder** [comment]
### Update

George with CTM informed the team that they do not want null gl codes in the system. They'd rather we update the fee status from active to inactive. Priya has already done this. Considering this, I will update the AC to no longer ask to remove the codes, instead just update the fee status. (Priya said she's already done this.)

**Comment by TracyLinder** [comment]
@Nadin-Nader we need to get the updates to the 3 active codes to prod ASAP please 🙏🏼

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been deployed to TEST.  We still have to wait for Kyle to UAT 14112. Once he does that, i will send 14112 to be deployed to PROD and then you cand send this one. I will let George know both ticket numbers so they can be pushed in order.

**Comment by TracyLinder** [comment]
### UAT Results - Failed on SCP ❌
FYI @Nadin-Nader @Lpriya-LIT 




**SCP - it doesn't look like this has been updated? ❌**
- **Test RSN is 13118707**

![image](https://github.com/cityofaustin/atd-data-tech/assets/49930868/a65e0b2b-ebb7-4fe7-821b-8e5c0d47ea0c)


**TURP - Usage Fee - Sidewalk/Behind Curb - ✅**

![image](https://github.com/cityofaustin/atd-data-tech/assets/49930868/3960f3a0-366c-47ba-bc01-01f541eb9759)

**Comment by TracyLinder** [comment]
### UAT Approved 
@Lpriya-LIT @Nadin-Nader please promote to PROD

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT @TracyLinder This has been sent to the DevOps CM Team to be deployed to PROD under **SCTASK0377029**

**Comment by TracyLinder** [comment]
Erin was informed. @Lpriya-LIT has all work on this card been completed?

**Comment by Lpriya-LIT** [comment]
@TracyLinder Yes, the 3 GL codes are updated and the active flags for the GL codes in the list are updated to N.
